src/main/js/controls/PairedResult/PairedResult.js

```diff
@@ -258,13 +258,43 @@ class PairedResult extends GraphContent {
      * @inheritdoc
      */
     reflow(graph, graphData) {
+        const eventHandlers = {
+            clickHandler: clickHandler(graph, this, graph.config, graph.svg),
+            hoverHandler: hoverHandler(graph.config, graph.svg)
+        };
+        const constructLegendLabels = (d, type) =>
+            Object.assign(
+                {},
+                {
+                    shape: getValue(d.shape, type),
+                    color: getValue(d.color, type),
+                    label: getValue(d.label, type),
+                    key: `${d.key}_${type}`,
+                    values: d.values,
+                    legendOptions: d.legendOptions,
+                    type
+                }
+            );
+        const reflow = !!this.config.values.length;
         this.config.values = graphData.values;
-        this.dataTarget = processDataPoints(graph.config, this.config, true);
+        this.dataTarget = processDataPoints(graph.config, this.config, reflow);
```

Comment:
Instead of creating a new variable, Can you directly use the value
```
this.dataTarget = processDataPoints(graph.config, this.config, !!this.config.values.length);

OR

this.dataTarget = processDataPoints(graph.config, this.config, this.config.values.length > 0);


````

Reply:
Modified [here](https://github.com/NarasimhaShenoi/carbon-graphs/commit/5543385fa2e773838ef3252436e5bef097053627)

Reply:
We cannot directly use that over here, we are checking the values in the config object before we update it with new incoming values in the line [279](https://github.com/cerner/carbon-graphs/pull/289/files/002f1d82b6fe0e97710680aaeb760b5ed0692794#diff-bcd050bbda8fed1d36ac3dc030996bcdR279). Reverted [here](https://github.com/NarasimhaShenoi/carbon-graphs/commit/0a77c55751c76ee55024f4a2a93e7bf727043a6b)